modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/beta_badge/BetaBadge.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayPopover, {ALIGN_POSITIONS} from '@clayui/popover';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import React from 'react';
+
+interface IProps {
+	tooltipAlign: typeof ALIGN_POSITIONS[number];
+}
+
+export default function BetaBadge({tooltipAlign = 'top'}: IProps) {
```

Comment:
We probably want to inline the interface here:

`
... BetaBadge({tooltipAlign = 'top'}: {tooltipAlign: typeof ALIGN_POSITIONS[number]}) {
`

Reply:
yep makes sense, I was not sure how many stuff I would need to pass to this thing, being the padding one of the items that made me doubt